.mailmap

```diff
@@ -0,0 +1,21 @@
+Adarsh Pyarelal <adarsh.pyarelal@gmail.com> Adarsh <adarsh.pyarelal@gmail.com> 
+Adarsh Pyarelal <adarsh.pyarelal@gmail.com> Adarsh Pyarelal <adarsh@email.arizona.edu>
+Albert Steppi <albert_steppi@hms.harvard.edu> Albert Steppi <albertsteppi@Alberts-MacBook-Pro.local>
+Albert Steppi <albert_steppi@hms.harvard.edu> steppi <albert_steppi@hms.harvard.edusdfjdifjijs>
+Albert Steppi <albert_steppi@hms.harvard.edu> steppi <albert_steppi@hms.harvard.edu>
+Ben Gyori <ben.gyori@gmail.com> Ben <ben@lspbgyorimbp.private.wireless.med.harvard.edu> 
+Ben Gyori <ben.gyori@gmail.com> bgyori <ben.gyori@gmail.com>
+Ben Gyori <ben.gyori@gmail.com> Benjamin M. Gyori <ben.gyori@gmail.com>
+Daniel Milstein <djmilstein@gmail.com> Daniel Milstein <djmilstein (at) gmail period com>
+Diana Kolusheva <dianakolusheva@gmail.com> Diana Kolusheva <35860871+dianakolusheva@users.noreply.github.com>
+Diana Kolusheva <dianakolusheva@gmail.com> dianakolusheva <dianakolusheva@gmail.com>
+Diana Kolusheva <dianakolusheva@gmail.com> kolusask <19ak97@gmail.com>
```

Comment:
This is not me :)

Reply:
oOoOoOo sorry, I just guessed because it shared a lot of letters of your surname